apps/xprof_gui/priv/app/call_tracer.jsx

```diff
@@ -256,7 +255,12 @@ export default class CallsTracer extends React.Component {
         const sortedItems = data.items.sort();
         const lastId = sortedItems.length === 0 ? this.state.offset : _.last(sortedItems).id;
         nextState.offset = lastId;
-        nextState.items = this.state.items.concat(sortedItems);
+        let itemsToAdd = sortedItems.filter(item => {
```

Comment:
Don't we have `lodash` there for a `.sort(...).uniq()` combo?

Reply:
Sorry for laziness, my change is completely unreadable ... I will rewrite this :)